envoy/scripts/copy_envoy_config.sh

```diff
@@ -6,29 +6,101 @@
 # Get the directory of the script
```

Comment:
Just as a heads up - I'm skipping this in leu of time but will edit when going through the E2E flow again.

Reply:
> Just as a heads up - I'm skipping this in leu of time but will edit when going through the E2E flow again.

SGTM but just FYI this file it not currently involved in the E2E at all. 

I started playing around with how we could implement a FULL E2E suite utilizing Tilt to spin up the full stack including Envoy but it's non-trivial and will have to wait until the new year. 

But just FYI I have been thinking about it and tentatively started prototyping.